createdisk.sh

```diff
@@ -25,6 +25,11 @@ VM_IP=$(arp -an | grep $(sudo virsh dumpxml ${CRC_VM_NAME} | grep '<mac' | grep
 # Remove audit logs
 ${SSH} core@${VM_IP} -- 'sudo find /var/log/ -iname "*.log" -exec rm -f {} \;'
 
+# Remove moby-engine pacakge
+# Shutdown and Start the VM after installing the hyperV daemon packages to have 
+# latest rpm-ostree deployment doesn't have this package
```

Comment:
Is this an intended change? Or an attempt to change the comment at line 42/47 which ended in the wrong place?